source/fundamentals/authentication/mechanisms.txt

```diff
@@ -152,26 +152,33 @@ The driver checks for your credentials in the following sources in order:
 4. AWS EC2 endpoint. For more information, see `IAM Roles for Tasks
    <https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html>`_.
 
+.. important::
+
+   The driver only reads the credentials from the first method that it detects
+   in the order of the preceding list. For example, if you specify your
```

Comment:
```suggestion
   in the order given by the preceding list. For example, if you specify your
```

Reply:
This suggestion seems more appropriate than the original, but I think the phrase is usually "as given by" so I'll use that.